includes/Core/Admin/Screens.php

```diff
@@ -152,14 +152,23 @@ function() {
 		add_filter(
 			'menu_order',
 			function( array $menu_order ) {
-				$new_order = array();
-				foreach ( $menu_order as $index => $item ) {
-					if ( 'index.php' === $item || 0 === strpos( $item, self::PREFIX ) ) {
-						$new_order[] = $item;
-						unset( $menu_order[ $index ] );
+				// Move the Site Kit dashboard menu item to be one after the index.php item if it exists.
+				$dashboard_index = array_search( 'index.php', $menu_order, true );
+
+				$sitekit_index = false;
+				foreach ( $menu_order as $key => $value ) {
+					if ( substr( $value, 0, strlen( self::PREFIX ) ) === self::PREFIX ) {
```

Comment:
Maybe simpler would be:
```suggestion
					if ( strpos( $value, self::PREFIX ) === 0 ) {
```